### ISSUE TITLE
Suggestion to include filtering by Prgram languge version

### ISSUE THREAD

**Issue by Aelkx** [Codewars/codewars.com#1018]
So, as the title suggest its a problem which I sometimes encounter using a newer version that some _katas_  are enocded for a older version, and sometimes (I learnt to check now) that I spend a few minutes tryning to figure out why the code doesn't work when I hadn't relised it did not support this version. 

Right now its kinda a mixed bag, I can select a _kata_ but I have no indication what version it will support, so I gotta just hope that it supports the my version.. if there can be added functionality for this that would be cool!

**Comment by ice1000** [comment]
It's `Programming Language`, not `Program Languege`

**Comment by Chrono79** [comment]
Any update on this?
I would be nice if you could either filter in kata list (maybe disable the filter when there is not a single language selected) by language version supported. I mean, if you only want to solve katas in Swift 4.0, or in Python 3.6. Or show the availables versions somewhere from outside the trainer.

**Comment by kazk** [comment]
Yeah, I've been wanting to improve the search too, but I don't think I have the time anytime soon. I'll update when anything changes.

I've been thinking of reworking the search to support search operators. Like `language:swift@4`, `language:python@>=3`, `intitle:word` (like Google), `indescription:word`, `author:name`, `progress:completed`, `progress:completed_in(python|ruby)`, `status:needs_approval` etc.

> Or show the availables versions somewhere from outside the trainer.

This might be low effort, so I'll look into it.
